CHANGELOG.md

```diff
@@ -1,4 +1,57 @@
+* compute: added `max_stream_duration` field to `google_compute_url_map` resource ([#9387](https://github.com/hashicorp/terraform-provider-google-beta/pull/9387))
+* compute: added `network_interface.network_attachment` field to `google_compute_instance` resource (ga) ([#9418](https://github.com/hashicorp/terraform-provider-google-beta/pull/9418))
+* compute: added `network_interface.network_attachment` to `google_compute_instance` data source (ga) ([#9418](https://github.com/hashicorp/terraform-provider-google-beta/pull/9418))
+* compute: added fields `architecture`, `source_instant_snapshot`, `source_storage_object`, `resource_manager_tags`  to `google_compute_disk`. ([#9412](https://github.com/hashicorp/terraform-provider-google-beta/pull/9412))
+* container: added enum  value `UPGRADE_INFO_EVENT` for GKE notification filter in `google_container_cluster` resource ([#9421](https://github.com/hashicorp/terraform-provider-google-beta/pull/9421))
+* iam: added `AZURE_AD_GROUPS_ID` field to `google_iam_workforce_pool_provider.extra_attributes_oauth2_client.attributes_type` resource ([#9433](https://github.com/hashicorp/terraform-provider-google-beta/pull/9433))
+* networkconnectivity: added `policy_mode` field to `google_network_connectivity_hub` resource ([#9409](https://github.com/hashicorp/terraform-provider-google-beta/pull/9409))
+* networkservices: added `location` field to `google_network_services_grpc_route` resource ([#9429](https://github.com/hashicorp/terraform-provider-google-beta/pull/9429))
+* storagetransfer: added `logging_config` field to `google_storage_transfer_job` resource ([#9378](https://github.com/hashicorp/terraform-provider-google-beta/pull/9378))
+
+BUG FIXES:
+* bigquery: updated the `max_staleness` field in `google_bigquery_table` to be a computed field ([#9411](https://github.com/hashicorp/terraform-provider-google-beta/pull/9411))
+* chronicle: fixed an error during resource creation with certain `run_frequency` configurations in `google_chronicle_rule_deployment` ([#9422](https://github.com/hashicorp/terraform-provider-google-beta/pull/9422))
+* discoveryengine: fixed bug preventing creation of `google_discovery_engine_target_site` resources ([#9436](https://github.com/hashicorp/terraform-provider-google-beta/pull/9436))
+* eventarc: fixed an issue where `google_eventarc_trigger` creation failed due to the region could not be parsed from the trigger's name ([#9383](https://github.com/hashicorp/terraform-provider-google-beta/pull/9383))
+* gemini: fixed permadiff on `product` field in `google_gemini_data_sharing_with_google_setting_binding` resource (Beta) ([#9437](https://github.com/hashicorp/terraform-provider-google-beta/pull/9437))
+* publicca: encode b64_mac_key in base64url, not in base64 ([#9424](https://github.com/hashicorp/terraform-provider-google-beta/pull/9424))
```

Comment:
```suggestion
* publicca: encoded `b64_mac_key` in base64url, instead of base64 in `google_public_ca_external_account_key` ([#9424](https://github.com/hashicorp/terraform-provider-google-beta/pull/9424))
```